HotelManagerment/com.spring.sample/src/com/spring/sample/controller/searchRoom.java

```diff
@@ -0,0 +1,21 @@
+package com.spring.sample.controller;
+
+import java.util.Locale;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+
+@Controller
+@EnableWebMvc
+public class searchRoom {
+	private static final Logger logger = LoggerFactory.getLogger(searchRoom.class);
+	@GetMapping(value = "/searchRoom")
+	public String add(Locale locale, Model model) {
+		logger.info("Requesting login form");
+		return "sessions/searchRoom/searchRoom";
```

Comment:
```suggestion
	@GetMapping(value = "/rooms")
	public String index(Locale locale, Model model) {
		logger.info("Requesting login form");
		return "sessions/rooms/index";
```

cái này là code màn hình list room nhỉ
đọc lại về coding convention liên quan đến tạo controller đi em